### DISCUSSION TITLE
Reading field values depending on the field type

### DISCUSSION THREAD

**Discussion by sabrinadeveloper2023** [pnp/pnpcore#1161] [Q&A]
Hi,

I need help on how I may read the listitem field values - based on the field type.
I was not able to identify examples or notes about this in the PnP Core Help. Any examples that can be shared with me are much appreciated.

Thanks!

**Comment by jansenbe** [comment]
Hi @sabrinadeveloper2023 , see https://pnp.github.io/pnpcore/using-the-sdk/listitems-fields.html#multi-user-fields for documentation on that topic
